internal/multirun.py

```diff
@@ -22,9 +22,9 @@ class Command(NamedTuple):
 
 def _run_command(command: Command, block: bool, **kwargs) -> Union[int, subprocess.Popen]:
     if platform.system() == "Windows":
-        bash = shutil.which("bash.exe")
+        bash = os.environ.get("BAZEL_SH") or shutil.which("bash")
```

Comment:
```suggestion
        bash = os.environ.get("BAZEL_SH") or shutil.which("bash.exe")
```

shouldn't this still have the `.exe`?

Reply:
well, yes, but `where bash` could also resolve to `bash.cmd` and `bash.bat` in case someone had a wrapper for it. 

it's an edge case so I think it's fine to keep `bash.exe`